pytorch_toolkit/image_classification/model_templates/custom-classification/README.md

```diff
@@ -1,15 +1,64 @@
 # Image classification
-
-Models that are able to classify images.
+## Description of the models
```

Comment:
I don't mind to change structure of README.md, but if you change it, please do it for all other templates.
I guess it can be remained as it is (in develop branch). But if you really want..

Reply:
Since that, can I just remove all additional headers, but remain the content?

Reply:
I think yes